_data/simple-icons.json

```diff
@@ -8200,8 +8200,8 @@
         },
         {
             "title": "Signal",
-            "hex": "2592E9",
-            "source": "https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms"
+            "hex": "3977F0",
```

Comment:
Thank you for the changes! Just to be sure before merging, where did you take this color?
The color from the SVG source is `#3A76F0`.

Reply:
from the website brand icon, here: https://signal.org/en/

Reply:
I also took it from the github signal svg logo linked as source, and it gives me the same `#3977F0` value.
Where is the `#3A76F0` coming from?

I use a Chrome plugin to get colors: chrome-extension://hmdcmlfkchdmnmnmheododdhjedfccka/popup.html#
Maybe the tool is responsible for the color difference?

Reply:
The color is embedded directly in the SVG:
![image](https://user-images.githubusercontent.com/12817388/129575778-8be89d4f-33f5-4978-9972-5320a48878c3.png)

The plugin might be responisible for the difference.

Reply:
That's bad news. I won't be able to trust the plugin moving forward.
Restored the hex to that color. Are we ready to merge this one?